src/main/java/org/broadinstitute/hellbender/Main.java

```diff
@@ -100,7 +92,9 @@ public Object instanceMain(final String[] args, final List<String> packageList,
      * This method is not intended to be used outside of the GATK framework and tests.
      */
     public Object instanceMain(final String[] args) {
-        return instanceMain(args, getPackageList(), getClassList(), getCommandLineName());
+        final GATKConf configuration = getConfiguration();
+        FeatureManager.setConfiguration(configuration);
```

Comment:
Rather than pass the full configuration object to engine classes like `FeatureManager`, I'd prefer to pass just the `List<String>` of packages to use for codec discovery.